src/main/java/org/libdohj/core/AltcoinSerializer.java

```diff
@@ -1,18 +1,14 @@
-    /*
- * To change this license header, choose License Headers in Project Properties.
- * To change this template file, choose Tools | Templates
- * and open the template in the editor.
- */
+/*
+* To change this license header, choose License Headers in Project Properties.
```

Comment:
Sorry, just spotted this; happy with the first line indentation fix, but could you put the space back in the rest so they line up please?